atm.js

```diff
@@ -1,3 +1,150 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+ //DEPOSIT MONEY FROM CHECKING
+
+
+
+  var checkingDeposit = $("#checking .deposit")
+  var savingsDeposit = $(“#savings .deposit”)
+
+
+  checkingDeposit.on("click", function(){
+
+
+    var checkingUserInput = $("#checking .input").val()
+    console.log("This is the user input:" + checkingUserInput)
+
+     create a variable that stores "checking"‘s balance(in this exercise = $0)
```

Comment:
This looks like it's supposed to be a comment but you didn't precede it with `//`. This will cause your code to break. Were you able to properly test your code?